### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by 0xf8f8ff**
- navigation bar always visible
- no empty `Settings` links
- striped tables
- two-button layout for lists with course members
- adjusted position for `Approve All` button

### CODE REVIEW

File: public/src/pages/views/MemberView.tsx [hunk 1]
```diff
@@ -123,44 +123,48 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
         </div>;
     }
 
-    public handleAction(userRel: IUserRelation, link: ILink) {
+    public async handleAction(userRel: IUserRelation, link: ILink) {
```
Comment (meling) [inline public/src/pages/views/MemberView.tsx]: This code was a bit long. Could these be split into separate methods for the different cases?

File: public/src/pages/views/MemberView.tsx [hunk 2]
```diff
@@ -82,11 +81,11 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
...
                 [],
-                ActionType.Menu,
+                ActionType.InRow,
                 (user: IUserRelation) => {
                     const links = [];
                     if (user.link.getStatus() === Enrollment.UserStatus.TEACHER) {
-                        links.push({ name: "This is a teacher", extra: "primary" });
+                        links.push({ name: "Teacher", extra: "light" });
                     } else {
                         links.push({ name: "Make Teacher", uri: "teacher", extra: "primary" });
```
Comment (meling) [inline public/src/pages/views/MemberView.tsx]: Suggest to replace `Make Teacher` with `Appoint as Teacher`.
Also, perhaps make a little space between the buttons, so they are not touching.

File: public/src/pages/views/MemberView.tsx [hunk 3]
```diff
@@ -82,11 +81,11 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
...
                         links.push({ name: "Reject", uri: "reject", extra: "danger" });
```
Comment (meling) [inline public/src/pages/views/MemberView.tsx]: Suggest to change `Reject` with `Remove` (in the Registered Users list). For the Pending Users we should still keep `Reject`.

File: public/src/pages/views/MemberView.tsx [hunk 4]
```diff
@@ -45,18 +45,17 @@ export class MemberView extends React.Component<IUserViewerProps, IUserViewerSta
...
-       }
+    public componentWillReceiveProps(newProps: IUserViewerProps) {
+        this.setState({
+            acceptedUsers: newProps.acceptedUsers,
+            rejectedUsers: newProps.rejectedUsers,
+            pendingUsers: newProps.pendingUsers,
+        });
     }
 
     public renderRejectedView() {
```
Comment (meling) [inline public/src/pages/views/MemberView.tsx]: Is this still used? (now that we reject users from a course... and delete the enrollment from the database.)

File: public/src/pages/views/MemberView.tsx (hunk 1 above)
Comment (0xf8f8ff) [reply public/src/pages/views/MemberView.tsx]: Done.

File: public/src/pages/views/MemberView.tsx (hunk 2 above)
Comment (0xf8f8ff) [reply public/src/pages/views/MemberView.tsx]: Are you sure about that? `Appoint as Teacher` button will take even more space.

File: public/src/pages/views/MemberView.tsx (hunk 3 above)
Comment (0xf8f8ff) [reply public/src/pages/views/MemberView.tsx]: Done.

File: public/src/pages/views/MemberView.tsx (hunk 2 above)
Comment (meling) [reply public/src/pages/views/MemberView.tsx]: I'm leaning towards changing it to `Promote` only in the button. Then we could have a tool tip on the button?? with `Promote to teacher`, if possible?

**Pull request merged**